doc/rbd/rbd-operations.rst

```diff
@@ -10,5 +10,6 @@
    Mirroring <rbd-mirroring>
    Live-Migration <rbd-live-migration>
    Persistent Cache <rbd-persistent-cache>
+   Persistent Write-back Cache <rbd-persistent-write-back-cache>
```

Comment:
I would merge it cleanly into the existing `Persistent Cache` document (or else rename the other one to reference the fact that it's a read-only cache to avoid confusion).

Reply:
I renamed 'persistent cache'.